src/components/forms/OnboardFormWithoutOrg.tsx

```diff
@@ -0,0 +1,104 @@
+import { Component, useContext, Show } from 'solid-js'
+import { Form } from 'solid-js-form'
+import * as Yup from 'yup'
+import { useNavigate } from 'solid-app-router'
+import { createMutation, createQuery } from 'solid-urql'
+
+import Input from '../Input'
+import Button from '../Button'
+import { AuthContext } from '../../context/AuthProvider'
+
+const CREATE_USER = `
+  mutation($input: CreateUserInput!) {
+    createUser(createUserInput: $input) {
+      id
+    }
+  }
+`
+
+const GET_INVITE = `
+  query($code: ID!) {
+    invite(code: $code) {
+      id
+      organization {
+        id
+        name
+      }
+    }
+  }
+`
+
+type Props = {
+  inviteCode: string
+}
+
+const OnboardFormWithoutOrg: Component<Props> = ({ inviteCode }) => {
+  const [{ user }] = useContext(AuthContext)
+  const [inviteResult] = createQuery({
+    query: GET_INVITE,
+    variables: { code: inviteCode },
+  })
```

Comment:
Should we have an expiring on these invite codes? i.e you can check the date.now compared to when it was created, if it is over 30 days, they need a new code. In cases where ppl leave and stuff. But might be overkill

Reply:
Just posting for visibility but we've decided this is a stretch feature.